database/repositories/helper.py

```diff
@@ -5,10 +5,11 @@
 
 
 class HelperRepository(Repository):
-    def __init__(self, sessionmaker: session, user_rank_model: Base, helper_range_model: Base):
+    def __init__(self, sessionmaker: session, user_rank_model: Base, helper_range_model: Base, helper_reward_model: Base):
         super(HelperRepository, self).__init__(sessionmaker=sessionmaker, model=user_rank_model)
         self.user_model = user_rank_model
         self.helper_model = helper_range_model
```

Comment:
jak już rozdzielacie na helper_range_model i helper_reward_model, to ta zmienna też mogłaby się nazywać helper_range_model tak jak niżej